### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by peabee**
Add S15Pup
Remove Slacko6
Remove TahrPup
Move VanillaPup to download page
Remove newsletters links

Update download page to match

### CODE REVIEW

**Review by 01micko — APPROVED** [review]

**Comment by peabee** [comment]
Thank you Mick.

File: download.md [hunk 1]
```diff
@@ -17,17 +17,23 @@ See: Official Pup build recipes at [Woof-CE on GITHUB]
 |XenialPup64     |UbuntuLTS16.04 compatible Official Pup (64bit)    |[Xenialpup64 Forum Page][ux64]|
 |Tahrpup         |UbuntuLTS14.04 compatible Official Pup (32bit)    |[Tahrpup Forum Page][ut32]    |
 |Tahrpup64       |UbuntuLTS14.04 compatible Official Pup (64bit)    |[Tahrpup64 Forum Page][ut64]  |
-|ScPup           |SlackwareCurrent compatible Pup (32 & 64bit)      |[ScPup Forum Page][slc]      |
+|S15Pup          |Slackware15.0 compatible Official Pup (32 & 64bit)|[S15Pup Forum Page][sl15]     |
```
Comment (dimkr) [inline download.md:20]: download.md is very hard to reach, the link is hidden pretty well. Putting a Puppy family distro in download.md is almost the same as removing it from the homepage.

Comment (dimkr) [inline download.md:20]: If S15Pup is newer than these old, outdated and insecure Puppy releases, why don't you put it at the top?

**Comment by peabee** [comment]
Changed menu so download goes to download page........

Always been in blocks of compatibility and then freshness within those blocks - Ubuntu seems to be more popular than Slackware so probably should come first.............

**Comment by dimkr** [comment]
> Always been in blocks of compatibility and then freshness

It's been this way, so it must be this way forever?

What's the point in announcing a new "official release" then, if it's not the recommended release for most new users and the first option they see?

**Review by dimkr — APPROVED** [review]
Not a fan of this discrimination against "unofficial" stuff, but the homepage is outdated and needs some cleanup work anyway.

**Pull request merged**